packages/echo-core/setupTests.js

```diff
@@ -7,3 +7,9 @@ Object.defineProperty(window, 'fetch', {
 Object.defineProperty(window, 'crypto', {
     value: { getRandomValues: mGetRandomValues }
 });
+
+jest.mock('../echo-base/src/errors/randomHelper', () => {
+    return {
+        randomId: jest.fn(() => 'ignore')
```

Comment:
Just to help better read possible erros in unit tests: would recomend to use a more specific mocked value here to the actual functionality mocked.

Maybe something like this:
'mocked-static-id-9999'